pkg/kube/watcher.go

```diff
@@ -52,7 +62,7 @@ func WatchForServices(
 	ctx context.Context,
 	configPath string,
 	k8sServiceListenerIP net.IP,
-	enablePrivilegedService bool,
+	listenerOnlyMode bool,
```

Comment:
While `enablePrivilegedService` is self-explanatory, `listenerOnlyMode` is not (and it doesn't match any of the four cases in the newly-updated module documentation either).

Reply:
Renamed to `enableListeners` as per our conversation.